windows/win_iis_virtualdirectory.ps1

```diff
@@ -102,7 +102,12 @@ try {
     $result.changed = $true
   }
 
-  $directory = Get-WebVirtualDirectory -Site $site -Name $name
+  $directory = if($application) {
```

Comment:
This works, but can you follow the arg splat pattern used above on New-WebVirtualDirectory instead of repeating the command?